package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@popperjs/core": "2.11.5",
     "classnames": "2.3.1",
+    "file-saver": "^2.0.5",
```

Comment:
This should be pinned. (But renovate will also fix that.)

Reply:
fixed